### PR TITLE
Improve formatting, English, and consistency

### LINKED ISSUE #114: Improve consistency and English
### Xmake Version

025fbe9

### Operating System Version and Architecture

macOS Ventura 13.2

### Describe Bug

Not a bug, but improve the consistency and grammar of some of the documentation. I understand English is not the author's first language, and am submitting this issue to attach a PR to.

### Expected Behavior

A clear reading experience, not:

> we need not write any make-like file (xmake.lua, makefile.am, cmakelist.txt, etc.) and also build it directly.

### Project Configuration

Not applicable

### Additional Information and Error Logs

Not applicable

### PR DESCRIPTION
**Pull request by IsaccBarker**
Fix #114.

Also, I was unsure whether to use `xmake`, `Xmake`, `XMake`, or `xMake`. All four are used in various places. I went with the second option, and corrected as I went, because that's the style of the first `Xmake` that occurred in the REAMDE.

### CODE REVIEW

File: about/introduction.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
+# Introduction
 
-xmake is a lightweight cross-platform build utility based on Lua. It uses xmake.lua to maintain project builds. Compared with makefile/CMakeLists.txt, the configuration syntax is more concise and intuitive. It is very friendly to novices and can quickly get started in a short time. Let users focus more on actual project development.
+Xmake is a lightweight cross-platform build utility based on Lua. It a Lua script to maintain project builds. Compared with Makefile/CMakeLists.txt, the configuration syntax is much concise and intuitive. As such, it's friendly to novices while still maintaining the flexibly required in a build system. With Xmake, you can focus on your project instead of the build.
```
Comment (waruqi) [inline about/introduction.md:3]: > It a Lua script to maintain

It is not just a lua script, it is a complete binary program. But it uses xmake.lua to maintain the project.

Comment (IsaccBarker) [reply about/introduction.md:3]: Ah, sorry. Didn't come across right then. I'll resolve it.

File: README.md [hunk 2]
```diff
@@ -479,7 +522,7 @@ $ xmake l
   }
 ```
 
-More builtin plugins exist, please see: [Builtin plugins](https://xmake.io/#/plugin/builtin_plugins)
+To see a list of bultin plugs, please visit [Builtin plugins](https://xmake.io/#/plugin/builtin_plugins).
```
Comment (infdahai) [inline README.md:525]: Not sure about using  `plugs` here. maybe `plugins`?

**Comment by waruqi** [comment]
Great work, thank you very much. 👍

File: README.md (hunk 2 above)
Comment (IsaccBarker) [reply README.md:525]: Sorry, typo on my part.

File: README.md [hunk 3]
```diff
@@ -5,137 +5,240 @@ Support this project by [becoming a sponsor](https://xmake.io/#/about/sponsor).
...
 
 ```bash
 wget https://xmake.io/shget.text -O - | bash
 ```
 
-#### via powershell
+### With PowerShell
 
-```powershell
+```sh
```
[the remark this answers is not in the document]
Comment (IsaccBarker) [reply README.md:58]: I don't think GitHub has [markdown support for PowerShell](https://github.com/jincheng9/markdown_supported_languages).

[the remark this answers is not in the document]
Comment (infdahai) [reply README.md:58]: you are right. I see the doc in Chinese version and it is also diabled.

**Pull request merged**